app/controllers/admin/organizations_controller.rb

```diff
@@ -83,19 +79,23 @@ def destroy
   private
 
   def organization_params
-    params.require(:organization).permit(
+    # Not sure why params.permit wasn't honoring this field for the native simple_form array
```

Comment:
This looks like a rails or simple_form bug to me. Probably because of the strange pluralization. It's just a theory, though.